keps/sig-node/2043-pod-resource-concrete-assigments/kep.yaml

```diff
@@ -16,7 +16,8 @@ reviewers:
   - "@dashpole"
 approvers:
   - "@sig-node-leads"
-prr-approvers: []
+prr-approvers:
```

Comment:
We're changing the process this cycle (this field is still around, but will be removed soon-ish).
But instead, please create a corresponding file under:
/keps/prod-readiness/sig-node/
directory

(see https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-storage/1412.yaml as a reference)

Reply:
Who is approver in this case? Still you or sig-node-leads?

Reply:
You need approval from both sig-node-leads and prr-approvers-team